assets/section-password.css

```diff
@@ -324,3 +324,29 @@ details.modal .modal__toggle-open {
   vertical-align: top;
   color: rgb(var(--color-foreground));
 }
+
+.background {
+  height: 100%;
+}
+
+.background-mobile {
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+@media only screen and (min-width: 750px) {
+  .background {
+    width: 100%;
+  }
+
+  .background-mobile {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 749px) {
+  .background:not(.background-mobile) {
+    display: none;
+  }
+}
```

Comment:
There seem to be an issue with links from the text rich text blocks. They're not using the right colors: 
![](https://screenshot.click/21-10-1uk5y-32brh.png)

It's not only for this sections, the original email signup one has it too. 

I think calling the `component-rte.css` should do the trick.

Reply:
Good catch! Fixed